tests/calculator-homework.spec.js

```diff
@@ -0,0 +1,197 @@
+const { test, expect } = require('@playwright/test');
+const {CalculatorStartPage} = require('../homework-pages/calculatorStartPage')
+const {CalculatorResultsPage} = require('../homework-pages/calculatorResultsPage');
+
+test.describe('Calculator test suite', () => {
+  let page;
+  test.beforeAll(async ({ browser }) => {
+    page = await browser.newPage();
+    startPage = new CalculatorStartPage(page);
+    resultsPage = new CalculatorResultsPage(page);
+  });
+
+  test.beforeEach(async () => {
+    await startPage.goto();
+  });
+
+
+  const buildsSumInt = ['Prototype', '1', '2', '3', '4', '5', '6', '7', '8', '9'];
+  buildsSumInt.forEach(buildSumInt => {
+    test(`${buildSumInt} build to check if calculating sum of integers`, async () => {
+      await startPage.selectBuild(buildSumInt);
+      await startPage.fillInFields('1', '2');
+      await startPage.selectOperationAndCalculate('Add');
+      const result = await resultsPage.getResult();
+      expect(result).toEqual('3');
+    });
+  });
+
+  const buildsSumPositive = ['Prototype', '1', '2', '3', '4', '5', '6', '7', '8', '9'];
+  buildsSumPositive.forEach(buildSumPositive => {
+    test.only(`${buildSumPositive} build to check if calculating sum of real positive numbers`, async () => {
+      await startPage.selectBuild(buildSumPositive);
+      await startPage.fillInFields('0.00000001', '0.00000001');
+      await startPage.selectOperationAndCalculate('Add');
+      const result = await resultsPage.getResult();
+      expect(result).toEqual('2e-8');
+    });
+  });
+
+  const buildsSumOnlyInt = ['Prototype', '1', '2', '3', '4', '5', '6', '7', '8', '9'];
+  buildsSumOnlyInt.forEach(buildSumOnlyInt => {
+    test(`${buildSumOnlyInt} build to check if calculating sum when result must be integer`, async () => {
+      await startPage.selectBuild(buildSumOnlyInt);
+      await startPage.fillInFields('1.2', '2.7');
+      await startPage.selectOperationAndCalculate('Add');
+      await page.click('#integerSelect');
+      const result = await resultsPage.getResult();
+      expect(result).toEqual('3');
+    });
+  });
+
+  const buildsSumNegative = ['Prototype', '1', '2', '3', '4', '5', '6', '7', '8', '9'];
+  buildsSumNegative.forEach(buildSumNegative => {
+    test.only(`${buildSumNegative} build to check if calculating sum of negative real numbers`, async () => {
+      await startPage.selectBuild(buildSumNegative);
```

Comment:
I see many places where you use these same 4-5 lines - select build, fill data, select operation, integers checkbox and get a result. These could be extracted into a separate function so tests would be cleaner.